app/posts/register-trainee-teachers/2021-11-22-service-updates.md

```diff
@@ -0,0 +1,60 @@
+
+
+As we added more training routes to Register and increased the amount of data collected, we experienced more support queries and more need for us to tell users about features we’d released.
+
+There was an obvious need for us to communicate with users about common support queries we were receiving. For example, on certain training routes there is no need for us to collect data on schools when a trainee is either self funded or is employed by a private school.
+
+We were making all users give us this data, so once we’d fixed it, we got back in touch with those who’d contacted us to let them know. But there was no way to know if it affected other users who may not have been in touch with us.
+
+## What we did
+
+Research participants mentioned the DTTP updates content, so we started looking at how we could implement this within Register. We felt it would be a low risk feature, but also give us another way to communicate with users without having to rely on emails.
+
+We looked at the [GOV.UK design system](https://design-system.service.gov.uk/) and their ‘What’s new’ content, featured on the start page.
+
+We did not want to solely focus on the what’s new aspect as this content could be used to give reminders to providers, for example, about census and outcome collection dates. We called it ‘News and updates’ to enable us to give various types of information to our users.
+
+Similar to the design system, the latest message is posted on Register’s sign in page. It contains the date the message was published, a headline and then main body content of the news or update.
+
+Beneath the latest message users can view all of the previous messages through the link - they do not have to be signed in to view this page. There’s also a link to this page from Register’s homepage:
+
+On the news and updates page, users can view all of the messages that have been posted, in chronological order. We’re using anchor links so it’s easier for users to find what they’re looking for. We can also link to specific posts in our other communications, for example, provider emails or the BAT (Becoming a teacher) newsletter.
+
+
+## Future considerations
+
+We’ve released this feature because it’s low risk and we’ll ask users about it in future research rounds. We’ll also monitor the visits to the ‘News and updates’ page to check if it’s being used.
+
+We will not use it for everything we release because the user interface should mostly fulfil that function. But when users have contacted us with common issues, we can address those who may not have contacted us about the same issue.
+
+We’ll also use it to remind users of significant dates and changes to ITT data collection, for example DTTP switch off and the ongoing work with HESA data.
+
+Anybody on the team can recommend an entry for ‘News and updates’. It will need to get approval from the product manager and then be written by the content designer before it's posted.
```

Comment:
```suggestion
Anybody on the team can recommend an entry for ‘News and updates’. It will need to get approval from the product manager and then be written by the content designer before it’s posted.
```